client/src/main/scala/drt/client/modules/GoogleAnalytics.scala

```diff
@@ -1,26 +1,82 @@
 package drt.client.modules
 
-
+import org.scalajs.dom
 import scala.scalajs.js
 import scala.scalajs.js.annotation.JSGlobalScope
 
 object GoogleEventTracker {
-  def isScriptLoaded: Boolean =js.Dynamic.global.analytics.isInstanceOf[js.Function]
-  def sendPageView(page:String):Unit={
-    if (isScriptLoaded) GoogleAnalytics.analytics("send","event","pageView", page)
+  val trackingCode: GoogleTrackingCode = GoogleTrackingCode.getTrackingCode(dom.document.location.href)
+  def port: String = dom.document.getElementById("port-code").getAttribute("value")
```

Comment:
can't we get this out of config rather than creating a hidden dom element?

Reply:
That would be cool if we can, can we?